src/pages/apply.tsx

```
@@ -444,43 +420,51 @@
+        errors={errors.longAnswerSocratica}
+        register={register}
+        currentLength={watch("longAnswerSocratica")?.split(/\s/g).length ?? 0}
       />
 
       <FormDivider label="Survey" />
 
       <FormInput
         label="What are your social media links?"
         id={"socialText"}
-        errors={errors.socialText}
+        errors={errors?.socialText && errors.socialText[0]}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)